packages/portal/backend/src/controllers/IntegrationController.ts

```diff
@@ -0,0 +1,43 @@
+import * as rp from "request-promise-native";
+import Config, {ConfigKey} from "../../../../common/Config";
+import Log from "../../../../common/Log";
+import {ClasslistTransport} from "../../../../common/types/PortalTypes";
+import Util from "../../../../common/Util";
+
+import {DatabaseController} from "./DatabaseController";
+import {GitHubActions, IGitHubActions} from "./GitHubActions";
+
+export class IntegrationController {
```

Comment:
I'd use a more descriptive name than `IntegrationController`. Maybe just `ClasslistAgent`? It's not clear what is being integrated here. It also might make more sense for this to just be in `util/` than in `controller/`.

Reply:
We went with ClasslistAgent.